src/Model/Networkproblem/Power/StochasticPQnode.cpp

```diff
@@ -15,6 +15,8 @@ namespace Model::Networkproblem::Power {
     Aux::schema::add_required(schema, "theta_P", Aux::schema::type::number());
     Aux::schema::add_required(schema, "sigma_Q", Aux::schema::type::number());
     Aux::schema::add_required(schema, "theta_Q", Aux::schema::type::number());
+    Aux::schema::add_required(
+        schema, "number_of_stochastic_steps", Aux::schema::type::number());
```

Comment:
This was actually quite tricky to find: the constructor was simply trying to access this attribute without checking whether or not it exists and since it was not defined in the schema, the default was not written into the schema, therefore the default was not applied and the attribute was missing.

While this was annoying I think this shows the strength of actually tying defaults into the schemas as it forces the schemas to be up to date for the defaults to work

Reply:
Oh yeah, sorry. This was a bad merge into master, should have left it in my branch. I will soon merge #223 , where this is done correctly.

I agree, that this failure is an advantage.